src/pages/workspaces/workspace/notebooks/AppLauncher.js

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash/fp'
+import { Fragment, useRef, useState } from 'react'
+import { div, h, iframe } from 'react-hyperscript-helpers'
+import * as breadcrumbs from 'src/components/breadcrumbs'
+import { PlaygroundHeader, StatusMessage } from 'src/components/cluster-common'
+import { Link, spinnerOverlay } from 'src/components/common'
+import { NewClusterModal } from 'src/components/NewClusterModal'
+import { Ajax } from 'src/libs/ajax'
+import * as Auth from 'src/libs/auth'
+import { handleNonRunningCluster } from 'src/libs/cluster-utils'
+import { reportError, withErrorReporting } from 'src/libs/error'
+import * as Utils from 'src/libs/utils'
+import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
+
+
+const AppLauncher = _.flow(
+  Utils.forwardRefWithName('AppLauncher'),
+  wrapWorkspace({
+    breadcrumbs: props => breadcrumbs.commonPaths.workspaceDashboard(props),
+    title: _.get('app'),
+    activeTab: 'notebooks'
```

Comment:
Suggestion: remove `activeTab` from this use of `wrapWorkspace`. I realize that this also applies to terminal but I think it's misleading to put RStudio logically under "Notebooks".